### PR TITLE
Partial fix for #8 - Another PR at Infern completes this fix

### PR DESCRIPTION
**Pull request by justinwilaby**

### CODE REVIEW

File: lib/addTemplatesToModule.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ function constructTemplateNode(t, node, isRoot) {
 				}
 				if (attrKeys.length > 0) {
 					template.push(t.ObjectProperty(t.identifier('attrs'), t.ObjectExpression(attrKeys.map(function (attrKey) {
-						return t.ObjectProperty(t.identifier(attrKey), getObjectValue(t, attrs[attrKey]));
+						var id = t.identifier("'" + attrKey + "'");
```
Comment (justinwilaby) [inline lib/addTemplatesToModule.js]: Added quotes for all attributes assignable to DOM Elements.

File: lib/index.js [hunk 2]
```diff
@@ -41,7 +41,8 @@ function processAttributes(t, attributes, root, templateElem) {
 			var attrName = attribute.name.name;
 
 			if (attribute.type === 'JSXAttribute') {
```
Comment (justinwilaby) [inline lib/index.js]: Attributes without a value, (as in all boolean attributes) will not have a `value` but must still enter the below flow without throwing

File: lib/index.js [hunk 3]
```diff
@@ -66,13 +67,17 @@ function processAttributes(t, attributes, root, templateElem) {
...
-				} else if (literalTypes[attribute.value.type]) {
+				} else if (literalTypes[attributeValue.type]) {
 					root.templateString += attrName + "=" + attribute.value.value + "|-|";
 					templateElem.attrs[attrName] = attribute.value.value;
-				} else if (attribute.value.type === "NullLiteral") {
+				} else if (attributeValue.type === "NullLiteral") {
 					root.templateString += attrName + "=null|-|";
 					templateElem.attrs[attrName] = null;
 				}
+				else if (!attributeValue.type){
```
Comment (justinwilaby) [inline lib/index.js]: This was done for lack of a better way to represent boolean attributes.

File: lib/addTemplatesToModule.js (hunk 1 above)
Comment (NoiSek) [inline lib/addTemplatesToModule.js]: Check that you need to add quotes first before you do? Minor, but you could save literally ones of precious, precious bytes.

**Comment by trueadm** [comment]
Thanks for the work on this @justinwilaby :) @NoiSek will this affect the `spike` branch too?

**Comment by justinwilaby** [comment]
@trueadm - This fix is ready to merge.

**Comment by trueadm** [comment]
Perfect

**Pull request merged**